src/pages/RealmScans/main.ts

```diff
@@ -0,0 +1,76 @@
+import { pageInterface } from '../pageInterface';
+
+export const RealmScans: pageInterface = {
+  name: 'RealmScans',
+  domain: 'https://realmscans.com',
+  languages: ['English'],
+  type: 'manga',
+  isSyncPage(url) {
+    return j.$('#screenmode').length > 0;
+  },
+  isOverviewPage(url) {
+    return utils.urlPart(url, 3) === 'series';
+  },
+  getImage() {
+    return j.$('div.thumb > img').attr('src');
+  },
+  sync: {
+    getTitle(url) {
+      return j.$('ol > li:nth-child(2) > a > span').text();
```

Comment:
With too specific I did not mean the classes. These are the less specific part. The `>` and tag types are more the Problem.

`.ts-breadcrumb li:nth-child(2) [itemprop="name"]`
or
`.ts-breadcrumb [href*="/series/"]`
for example work better here